controllers/v1/case/index.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+module.exports = async (fastify, opts) => {
+  const Case = fastify.sequelize.import('../../../db/models/history.js');
+
+  fastify.post('/case', { schema: { body: fastify.schemas().createCase } }, async (request, reply) => {
+    try {
+      const { postalCode, geo, condition, timestamp, symptoms } = request.body
```

Comment:
Is the request.body automatically validated by specifying the schema?

Reply:
Yep, it is.